pylib/Utilities/ExecuteCmd.py

```diff
@@ -118,19 +119,19 @@ def execute(self, options, cmdargs, testDef):
                     if fd == p.stdout.fileno():
                         read = p.stdout.readline()
                         if read:
-                            read = read.rstrip()
-                            testDef.logger.verbose_print('stdout: ' + read.decode("utf-8", errors='replace'))
+                            read = read.rstrip().decode('utf-8', errors='replace')
+                            testDef.logger.verbose_print('stdout: ' + read)
                             if merge:
-                                stderr.append(read.decode("utf-8", errors='replace'))
+                                stderr.append(read)
                             else:
-                                stdout.append(read.decode("utf-8", errors='replace'))
+                                stdout.append(read)
                             stdout_done = False
                     elif fd == p.stderr.fileno():
                         read = p.stderr.readline()
                         if read:
-                            read = read.rstrip()
-                            testDef.logger.verbose_print('stderr: ' + read.decode("utf-8", errors='replace'))
-                            stderr.append(read.decode("utf-8", errors='replace'))
+                            read = read.rstrip().decode('utf-8', errors='replace')
```

Comment:
Hi @jsquyres ,
I checked the PR and I it looks good to me. The code looks more clean.
I saw you didn't actually take out the utf-8 decoding though, and instead just moved it to another location.

Reply:
In this case, yes, because reading from stdout/stderr results in a byte stream, not a string.  `decode()` here will convert those bytes into a Python string.